cairo/src/lib.rs

```diff
@@ -228,8 +228,8 @@ pub use crate::{
     enums::*,
     error::{BorrowError, Error, IoError, Result},
     font::{
-        FontExtents, FontFace, FontOptions, FontSlant, FontType, FontWeight, Glyph, ScaledFont,
-        TextCluster, TextExtents, UserFontFace,
```

Comment:
I would've said that this should get a bugfix release but considering that nobody noticed the lack of those types in the public API for *years*....